client/data/marketplace/use-wpcom-plugins-query.ts

```diff
@@ -37,6 +37,17 @@ const fetchWPCOMPlugins = ( type: Type, searchTerm?: string, tag?: string ) => {
 	);
 };
 
+export const getFetchWPCOMPlugins = (
+	enabled: boolean,
+	type: Type,
+	searchTerm?: string,
+	tag?: string
+): [ QueryKey, QueryFunction< { results: Array< { [ plugin: string ]: Plugin } > } > ] => {
+	const cacheKey = getCacheKey( type + searchTerm + tag + `${ enabled ? 'enabled' : 'disabled' }` );
```

Comment:
Why would the `enabled` flag be a part of the query key? The `enabled` flag is used to make the `useQuery` hook into a "noop" conditionally, as the hooks itself must always be called.

Reply:
That makes sense. I copied over this way from where it was in `useWPCOMPlugins` and avoided tinkering with the details.